house/migrations/0001_initial.py

```diff
@@ -34,13 +34,80 @@ class Migration(migrations.Migration):
-                ('parent_profession_2', models.CharField(choices=[('Teacher', 'Teacher'), ('Student', 'Student'), ('Programmer', 'Programmer'), ('Artist', 'Artist'), ('Manager', 'Manager'), ('Army', 'Army'), ('Police', 'Police'), ('Doctor', 'Doctor'), ('Vet', 'Vet'), ('Nurse', 'Nurse'), ('Technichian', 'Technichian'), ('Cleaner', 'Cleaner'), ('Other', 'Other'), ('Unemployed', 'Unemployed')], default='Other', max_length=50)),
-                ('income', models.IntegerField(blank=True, null=True, validators=[django.core.validators.MinValueValidator(0)])),
-                ('children', models.IntegerField(blank=True, null=True, validators=[django.core.validators.MinValueValidator(0)])),
+                (
+                    'parent_profession_1',
+                    models.CharField(
+                        choices=[
+                            ('Teacher', 'Teacher'),
+                            ('Student', 'Student'),
+                            ('Programmer', 'Programmer'),
+                            ('Artist', 'Artist'),
+                            ('Manager', 'Manager'),
+                            ('Army', 'Army'),
+                            ('Police', 'Police'),
+                            ('Doctor', 'Doctor'),
+                            ('Vet', 'Vet'),
+                            ('Nurse', 'Nurse'),
+                            ('Technichian', 'Technichian'),
+                            ('Cleaner', 'Cleaner'),
+                            ('Other', 'Other'),
+                            ('Unemployed', 'Unemployed'),
+                        ],
+                        default='Other',
+                        max_length=50,
+                    ),
+                ),
+                (
+                    'parent_profession_2',
+                    models.CharField(
+                        choices=[
+                            ('Teacher', 'Teacher'),
+                            ('Student', 'Student'),
+                            ('Programmer', 'Programmer'),
+                            ('Artist', 'Artist'),
+                            ('Manager', 'Manager'),
+                            ('Army', 'Army'),
+                            ('Police', 'Police'),
+                            ('Doctor', 'Doctor'),
+                            ('Vet', 'Vet'),
+                            ('Nurse', 'Nurse'),
+                            ('Technichian', 'Technichian'),
+                            ('Cleaner', 'Cleaner'),
+                            ('Other', 'Other'),
+                            ('Unemployed', 'Unemployed'),
+                        ],
+                        default='Other',
+                        max_length=50,
+                    ),
+                ),
+                (
+                    'income',
+                    models.IntegerField(
```

Comment:
the same comment like the other migrations file :)

Reply:
I agree, lint comments on migration files are irrelevant.